src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -166,7 +166,7 @@ public boolean isAACLC() {
 	/**
 	 * @return True if the audio codec is AAC HE.
 	 */
-	public boolean isAACHE() {
+	public boolean isHEAAC() {
```

Comment:
You are the one that make mess in your PR, i didn't that when i approved.
So i kept that one.